src/components/Geogebra/GeogebraViews.js

```diff
@@ -0,0 +1,86 @@
+import GeogebraAdminView from './GeogebraAdminView';
+import feathersClient from '../../feathers-client';
+
+const api = feathersClient;
+
+export default class {
+    constructor(groups, params) {
+        this.groups = groups;
+
+        this.params = {
+            width: 300,
+            height: 300,
+            perspective: 'G',
+            // showAlgebraInput: false,
+            showToolBarHelp: false,
+            showMenubar: false,
+            enableLabelDrags: false,
+            showResetIcon: false,
+            showToolbar: false,
+            allowStyleBar: false,
+            useBrowserForJS: true,
+            enableShiftDragZoom: true,
+            errorDialogsActive: true,
+            enableRightClick: false,
+            enableCAS: false,
+            enable3d: false,
+            isPreloader: false,
+            screenshotGenerator: false,
+            preventFocus: true,
+            ...params,
+        };
+
+        this.multiParams = this.groups.map(g => ({
+            // eslint-disable-next-line no-underscore-dangle
+            container: `ggb_applet_${g._id}`,
+            // eslint-disable-next-line no-underscore-dangle
+            id: `ggb_applet_${g._id}`,
+            // eslint-disable-next-line no-underscore-dangle
+            groupId: g._id,
+            ...this.params,
+        }));
+
+        this.GAVs = this.multiParams.map(p => new GeogebraAdminView(p, p.groupId));
+
+        // eslint-disable-next-line no-underscore-dangle
+        this.workshopIds = this.groups.map(g => g._id);
+    }
+
+    inject() {
+        for (let i = 0; i < this.GAVs.length; i += 1) {
```

Comment:
Use `for/of` instead of raw `for`.

Reply:
Attempts to do so give a linter error : 

"iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations"

Reply:
Ignore it.